From cc21d7cb9d61eee4efef76e59a04257f1690c90e Mon Sep 17 00:00:00 2001
From: MonsieurApple <tony@oc.tc>
Date: Tue, 9 Jul 2013 09:48:49 -0500
Subject: [PATCH] Fix potion effects not being reset on death

Fix potion effects not being reset on death. Fixes BUKKIT-4515

diff --git a/src/main/java/net/minecraft/server/EntityPlayer.java b/src/main/java/net/minecraft/server/EntityPlayer.java
index 55a0612..5c9babf 100644
--- a/src/main/java/net/minecraft/server/EntityPlayer.java
+++ b/src/main/java/net/minecraft/server/EntityPlayer.java
@@ -922,7 +922,7 @@ public class EntityPlayer extends EntityHuman implements ICrafting {
         this.expTotal = this.newTotalExp;
         this.exp = 0;
         this.deathTicks = 0;
-        this.aG(); // Should be removeAllEffects.
+        this.aJ(); // Should be removeAllEffects.
         this.updateEffects = true;
         this.activeContainer = this.defaultContainer;
         this.killer = null;
-- 
1.8.1.2

